PurchasesCoreSwift/SubscriberAttributes/SubscriberAttribute.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  SubscriberAttribute.swift
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 7/1/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+@objc(RCSubscriberAttribute) public class SubscriberAttribute: NSObject {
+
+    @objc public static let keyKey = "key"
+    @objc public static let valueKey = "value"
+    @objc public static let setTimeKey = "setTime"
+    @objc public static let isSyncedKey = "isSynced"
+
```

Comment:
these properties should be private
and we could make them an extension of string or an enum for readability

Reply:
Negative ghostrider, these have to be public since `Purchases` is still a thing that references them temporarily. Added a TODO.

Reply:
`Purchases` references the `properties`, not the keys in the dict, though, these keys were only previously defined in the .m https://github.com/RevenueCat/purchases-ios/blob/main/Purchases/SubscriberAttributes/RCSubscriberAttribute.m#L13

Reply:
It was, until: https://github.com/RevenueCat/purchases-ios/pull/618/files#diff-d44094299a065a5dac814d5df51df093bb3626464295293fd4f441a2dc874cdeR317

Reply:
oohhh I see

Reply:
Yeah, so many moving pieces. I can't wait until everything has been moved and then we can do a pass on it like "Hey, does this thing here even make sense anymore?"